compiler/ballerina-backend-jvm-old/src/main/ballerina/compiler_backend_jvm/jvm_observability_gen.bal

```diff
@@ -20,6 +20,13 @@ function emitStopObservationInvocation(jvm:MethodVisitor mv, int strandIndex) {
         io:sprintf("(L%s;)V", STRAND), false);
 }
 
+function emitReportErrorInvocation(jvm:MethodVisitor mv, int strandIndex, int errorIndex) {
+    mv.visitVarInsn(ALOAD, strandIndex);
+    mv.visitVarInsn(ALOAD, errorIndex);
+    mv.visitMethodInsn(INVOKESTATIC, "org/ballerinalang/jvm/observability/ObserveUtils", "reportError",
```

Comment:
Better to move these class names to constants. Let's do it in a separate PR.

Reply:
Will send another PR